* ERROR: Execution of event handler 'uninative_event_fetchloader' failed
@ 2023-02-22 12:17 Murali Krishna
  0 siblings, 0 replies; only message in thread
From: Murali Krishna @ 2023-02-22 12:17 UTC (permalink / raw)
  To: yocto


[-- Attachment #1.1: Type: text/plain, Size: 5992 bytes --]

Hi Team Yocto,
Hi Team Yocto,

Greetings Of the Day.


i would like to build* YOCTO Image* for My Target Machine i.e.,*Kontron
Smarc-Samx6i.* The following link will provide complete information about
my Target Machine.
https://www.kontron.com/en/products/smarc-samx6i/p89810.

So here I have,

*--> BSP for My Target Machine *
*(successfully built yocto image in ubuntu14 and yocto-release is
yocto-pyro).*

*Now I want to migrate this BSP into the Dunfell Version*.
1.Created layer using bitbake
2.copied recipes from pyro layer
3. start with bitbake build Engine as bitbake-layers show-layers.(my bsp
layer added)
4.started to build yocto Image as bitbake *kontron-eval-image* then an
error arose.
................................................................................................................................................................................................................................................................................

murali@ubuntu:~/test/track_build$ bitbake kontron-eval-image
Loading cache: 100%
|#######################################################################################################################################################################|
Time: 0:00:00
Loaded 3775 entries from dependency cache.
NOTE: Resolving any missing task queue dependencies

Build Configuration:
BB_VERSION           = "1.46.0"
BUILD_SYS            = "x86_64-linux"
NATIVELSBSTRING      = "ubuntu-20.04"
TARGET_SYS           = "arm-fslc-linux-gnueabi"
MACHINE              = "smarc-samx6i"
DISTRO               = "fslc-x11"
DISTRO_VERSION       = "3.1"
TUNE_FEATURES        = "arm vfp cortexa9 neon thumb callconvention-hard"
TARGET_FPU           = "hard"
meta
meta-poky            = "dunfell:7d78d5f6e96eea26b98958a7b413f6c0743086b7"
meta-oe
meta-multimedia
meta-xfce            = "dunfell:e707e9b7cf5c62bff4fee029965a87b22dd4ccba"
meta-freescale       = "dunfell:325c08c0773a2f1c54d28ea8aeda7c3d110d88a3"
meta-freescale-3rdparty = "dunfell:f37f72a85ba273b27347169b727c889f42b91d6c"
meta-freescale-distro = "dunfell:5d882cdf079b3bde0bd9869ce3ca3db411acbf3b"
meta-qt5             = "dunfell:5ef3a0ffd3324937252790266e2b2e64d33ef34f"
meta-gnome
meta-python
meta-networking      = "dunfell:e707e9b7cf5c62bff4fee029965a87b22dd4ccba"
meta-boundary        = "<unknown>:<unknown>"

ERROR: Execution of event handler 'uninative_event_fetchloader' failed
Traceback (most recent call last):
  File "/usr/lib/python3.8/os.py", line 213, in
makedirs(name='/opt/freescale/yocto/imx//uninative/b110bf2e10fe420f5ca2f3ec55f048ee5f0a54c7e34856a3594e51eb2aea0570',
mode=511, exist_ok=False):
             try:
    >            makedirs(head, exist_ok=exist_ok)
             except FileExistsError:
  File "/usr/lib/python3.8/os.py", line 213, in
makedirs(name='/opt/freescale/yocto/imx//uninative', mode=511,
exist_ok=False):
             try:
    >            makedirs(head, exist_ok=exist_ok)
             except FileExistsError:
  File "/usr/lib/python3.8/os.py", line 213, in
makedirs(name='/opt/freescale/yocto/imx', mode=511, exist_ok=False):
             try:
    >            makedirs(head, exist_ok=exist_ok)
             except FileExistsError:
  File "/usr/lib/python3.8/os.py", line 213, in
makedirs(name='/opt/freescale/yocto', mode=511, exist_ok=False):
             try:
    >            makedirs(head, exist_ok=exist_ok)
             except FileExistsError:
  File "/usr/lib/python3.8/os.py", line 223, in
makedirs(name='/opt/freescale', mode=511, exist_ok=False):
         try:
    >        mkdir(name, mode)
         except OSError:
PermissionError: [Errno 13] Permission denied: '/opt/freescale'

ERROR: Command execution failed: Traceback (most recent call last):
  File "/home/murali/test/sources/poky/bitbake/lib/bb/command.py", line
111, in runAsyncCommand
    commandmethod(self.cmds_async, self, options)
  File "/home/murali/test/sources/poky/bitbake/lib/bb/command.py", line
576, in buildTargets
    command.cooker.buildTargets(pkgs_to_build, task)
  File "/home/murali/test/sources/poky/bitbake/lib/bb/cooker.py", line
1481, in buildTargets
    bb.event.fire(bb.event.BuildStarted(buildname, ntargets),
self.databuilder.mcdata[mc])
  File "/home/murali/test/sources/poky/bitbake/lib/bb/event.py", line 214,
in fire
    fire_class_handlers(event, d)
  File "/home/murali/test/sources/poky/bitbake/lib/bb/event.py", line 121,
in fire_class_handlers
    execute_handler(name, handler, event, d)
  File "/home/murali/test/sources/poky/bitbake/lib/bb/event.py", line 93,
in execute_handler
    ret = handler(event)
  File "/home/murali/test/sources/poky/meta/classes/uninative.bbclass",
line 49, in uninative_event_fetchloader
    bb.utils.mkdirhier(tarballdir)
  File "/home/murali/test/sources/poky/bitbake/lib/bb/utils.py", line 747,
in mkdirhier
    raise e
  File "/home/murali/test/sources/poky/bitbake/lib/bb/utils.py", line 744,
in mkdirhier
    os.makedirs(directory)
  File "/usr/lib/python3.8/os.py", line 213, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/usr/lib/python3.8/os.py", line 213, in makedirs
    makedirs(head, exist_ok=exist_ok)
  File "/usr/lib/python3.8/os.py", line 213, in makedirs
    makedirs(head, exist_ok=exist_ok)
  [Previous line repeated 1 more time]
  File "/usr/lib/python3.8/os.py", line 223, in makedirs
    mkdir(name, mode)
PermissionError: [Errno 13] Permission denied: '/opt/freescale'


Summary: There were 2 ERROR messages shown, returning a non-zero exit code.
.............................................................................................................................................................................................................................................................................................
 here I'm attaching The screenshots of Related Error.
How to solve the above Error?

[-- Attachment #1.2: Type: text/html, Size: 7767 bytes --]

[-- Attachment #2: 12.png --]
[-- Type: image/png, Size: 122768 bytes --]

[-- Attachment #3: 11.png --]
[-- Type: image/png, Size: 94453 bytes --]

[-- Attachment #4: 10.png --]
[-- Type: image/png, Size: 194047 bytes --]

^ permalink raw reply	[flat|nested] only message in thread
only message in thread, other threads:[~2023-03-01 13:49 UTC | newest]

Thread overview: (only message) (download: mbox.gz / follow: Atom feed)
-- links below jump to the message on this page --
2023-02-22 12:17 ERROR: Execution of event handler 'uninative_event_fetchloader' failed Murali Krishna

